filament/backend/src/webgpu/WebGPUDriver.cpp

```diff
@@ -923,6 +925,17 @@ void WebGPUDriver::commit(Handle<HwSwapChain> sch) {
     assert_invariant(mCommandBuffer);
     mCommandEncoder = nullptr;
     mQueue.Submit(1, &mCommandBuffer);
+
+    static bool firstRender = true;
+    // For the first frame rendered, we need to make sure the work is done before presenting or we
+    // get a purple flash
+    if (firstRender) {
+        auto f = mQueue.OnSubmittedWorkDone(wgpu::CallbackMode::WaitAnyOnly,
+                [=](wgpu::QueueWorkDoneStatus) {});
+        const wgpu::Instance instance = mDevice.GetAdapter().GetInstance();
+        instance.WaitAny(f, std::chrono::duration_cast<std::chrono::nanoseconds>(1s).count());
```

Comment:
(from above comment) 
we already have `std::chrono::nanoseconds` why not `std::chrono::seconds` as well for the `1s`?
```suggestion
        instance.WaitAny(f, std::chrono::duration_cast<std::chrono::nanoseconds>(std::chrono::seconds(1)).count());
```
and then remove the `using namespace std::chrono_literals;` ?